deploy/charts/emqx-enterprise/templates/StatefulSet.yaml

```diff
@@ -121,6 +121,10 @@ spec:
           {{- end }}
           - name: ekka
             containerPort: 4370
+          - name: genrpc-manual
+            containerPort: 5369
+          - name: genrpc2-auto
+            containerPort: 5370
```

Comment:
there should be no need for this. ?

Reply:
we need it for port port_discovery = stateless in gen_rpc, yes this is confusion.

Reply:
https://github.com/emqx/gen_rpc/blob/46dd22100391f3d87daca989cde7e2ecc52a17b5/README.md?plain=1#L117

Reply:
with the override set in docker entrypoint, i don't think we need this port.